components/newsroom/FeaturedBlogPost.tsx

```diff
@@ -43,7 +43,7 @@ export default function FeaturedBlogPost({ post, className = '' }: FeaturedBlogP
   return (
     <div className={`rounded-lg ${className}`}>
       <article className='h-full rounded-lg'>
-        <Link href={post.slug}>
+        <div >
```

Comment:
Why do we need this change?

Reply:
The Link was causing the hydration to fail (see screenshot) 

![image](https://github.com/user-attachments/assets/eaa188c3-1c19-4359-ba67-884e95d0a080)


The reason behind the hydration error was that we are nesting a `<Link>` tag within `<Link>` tag. 

### The first `<Link>` Tag: 
https://github.com/asyncapi/website/blob/615930638c2610a358616650c821ef560d602790/components/newsroom/FeaturedBlogPost.tsx#L46

### The second (nested) `<Link>` Tag: 
https://github.com/asyncapi/website/blob/615930638c2610a358616650c821ef560d602790/components/newsroom/FeaturedBlogPost.tsx#L69-L78

Both of these links has the same target (i.e. post.slug). So removing the outer Ling tag does not have any side effects.